FirebaseAuth/Tests/Unit/FIRUserTests.m

```diff
@@ -667,7 +666,7 @@ - (void)testUserPropertiesAndNSSecureCoding {
                                              // Verify FIRMultiFactorInfo properties.
                                              XCTAssertEqualObjects(
                                                  user.multiFactor.enrolledFactors[0].factorID,
-                                                 FIRPhoneMultiFactorID);
+                                                 FIRPhoneMultiFactorInfo.FIRPhoneMultiFactorID);
```

Comment:
This looks like a public API change from a global string that should be noted if not already in the spreadsheet

Reply:
Added.

Reply:
And I did a scan for any new API breaks introduced and I didn't see any. Thanks for the reviews Paul, I'll go ahead and merge.